src/components/sidebar/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import Button from "@material-ui/core/Button";
+import Drawer from "@material-ui/core/Drawer";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import AccountCircleIcon from "@material-ui/icons/AccountCircle";
+import "./style.css";
+
+function Profile(props) {
+  // AccountCircleIcon can be replaced by Avatar when we add Profile Picture functionality
+  return (
+    <Button color="primary" startIcon={<AccountCircleIcon />}>
+      {props.name}
+    </Button>
+  );
+}
+
+function StreamItem(props) {
+  return (
+    <ListItem button onClick={() => alert("clicked " + props.streamName)}>
+      {props.streamName}
+    </ListItem>
+  );
+}
+
+function StreamsList(props) {
+  let list = new Array();
+  for (let stream of props.streams) {
+    list.push(<StreamItem streamName={stream} />);
+  }
```

Comment:
This is generally considered a better way of looping over arrays in JS - clear and succint:

```suggestion
  const list = props.stream.map(stream => <StreamItem streamName={stream} />);
```